addons/auth_passkey/lib/duo_labs/webauthn/INFO

```diff
@@ -0,0 +1,6 @@
+Modified imports to be compatible, otherwise the code is the same as v2.0.0
+https://github.com/duo-labs/py_webauthn/releases/tag/v2.0.0
+
+We were not able to add the package to our requirements for the following reasons:
+- Their requirements.txt required pyOpenSSL 23.3.0, which is not compatible with our requirements.
+- The package is not in the debian apt repositories.
```

Comment:
I think it is time to determine if we want to import the entire library or just the few parts which interests us.